services/eclipse-marketplace/eclipse-marketplace-base.js

```diff
@@ -1,13 +1,6 @@
 import { BaseXmlService } from '../index.js'
 
 export default class EclipseMarketplaceBase extends BaseXmlService {
-  static buildRoute(base) {
-    return {
-      base,
-      pattern: ':name',
```

Comment:
The downloads badge now has a different pattern; this helper didn't really bring much value anyway, I decided to get rid of it.